CHANGES_NEXT_RELEASE

```diff
@@ -13,3 +13,4 @@
 - Fix: MQTT disconnection when MQTT publish fail (so re-connection is forced at next MQTT notification)
 - Fix: lineMaxSize and infoPayloadMaxSize in the log admin REST API (#3707)
 - Fix: check for service database name too long at cache sync logic (#2848)
+- Fix: pagination support in request query forwarding to Context Providers (#4149, #847)
```

Comment:
@mapedraza reviewed time ago all the issues related with CPrs and forwarding. It would be great to have his feedback, just in case some other issue should be mentioned here.

Reply:
Added a new issue ( #4337 ) containing the list of all issues related with CPrs. Adding this issue to the CNR should help as @fgalan mentioned above

```suggestion
- Fix: pagination support in request query forwarding to Context Providers (#4149, #847, #4337)
```

Reply:
Fixed in [c7e89b0](https://github.com/telefonicaid/fiware-orion/pull/4275/commits/c7e89b028695297ef0f997fd69306dd468ae5c05)